tools/mkmapdata/regridbatch.sh

```diff
@@ -81,22 +81,19 @@ for res in $resols; do
        echo "regional"
        # For regional and (especially) single-point grids, we can get
        # errors when trying to use multiple processors - so just use 1.
```

Comment:
The changes in this file were discussed originally in this post:
https://github.com/ESCOMP/CTSM/pull/823#issuecomment-582951237